package-lock.json

```diff
@@ -1,8 +1,3938 @@
 {
   "name": "@nextcloud/router",
   "version": "2.0.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```

Comment:
what are you sneaking in?

Reply:
AH, it seems the lockfile was not updated. I did an npm install to test locally.
Seeing your engine uses npm 7 or 8, this is valid then. Are you ok to leave it there?

Reply:
reminder, npm 6 and under = lockfile version 1
npm 7 and above = lockfile version 2

Reply:
```
npm WARN old lockfile 
npm WARN old lockfile The package-lock.json file was created with an old version of npm,
npm WARN old lockfile so supplemental metadata must be fetched from the registry.
npm WARN old lockfile 
npm WARN old lockfile This is a one-time fix-up, please be patient...
npm WARN old lockfile 
```

Reply:
Otherwise I can split it :)

Reply:
:rocket:

Reply:
https://github.com/nextcloud/nextcloud-router/pull/436